CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(NEURON C CXX)
 # =============================================================================
 set(PROJECT_VERSION_MAJOR 8)
 set(PROJECT_VERSION_MINOR 0)
-set(CMAKE_CXX_STANDARD 98)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This fixed the problem I was having with auto&. Thanks!

Reply:
@nrnhines : note that we have to check autotools config to insert necessary C++11 compiler flag.

Reply:
@pramodk  The m4 package is self-sufficient for error handling, no need to document